src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -595,7 +623,36 @@ private String doRequest(HttpMethod httppost) throws IOException {
                 // 204, no content
                 return "";
             }
-            String response = new String(httppost.getResponseBody(), "UTF-8");
+            String response;
+            long len = -1L;
+            Header[] headers = httppost.getResponseHeaders("Content-Length");
```

Comment:
Cannot because this one is a `HttpMethod` that doesn't have the `getResponseContentLength()` method